doc/nrf/ug_nrf52.rst

```diff
@@ -211,41 +211,40 @@ FOTA upgrades can be used to replace the application.
 .. note::
    |note|
 
-FOTA over Bluetooth LE
+FOTA over Bluetooth Low Energy
 ======================
 
-To perform a FOTA upgrade, complete the following steps:
+To enable support for FOTA upgrades, do the following:
+
+   * Use :doc:`mcuboot:index-ncs` as the upgradable bootloader (:kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` must be enabled).
```

Comment:
Consider that :doc:`mcuboot:index-ncs`  is named "Introduction to MCUboot", so the sentence structure should accomodate the link accordingly